code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -48,57 +48,47 @@
 
 	handcuffs_apply(M, user)
 
-//Our inventory procs should be able to handle the following, but our inventory code is hot spaghetti bologni, so here we go
+//Our inventory procs should be able to handle the following, but our inventory code is hot spaghetti bologni, so here we go //There's no real reason for this to be a separate proc now but whatever
 /obj/item/weapon/handcuffs/proc/handcuffs_apply(var/mob/living/carbon/C, var/mob/user, var/clumsy = FALSE)
 	if(!istype(C)) //Sanity doesn't hurt, right ?
 		return FALSE
 
 	if(ishuman(C))
-		var/obj/effect/equip_e/human/O = new /obj/effect/equip_e/human()
-		O.source = user
-		O.target = C
-		O.t_loc = C.loc
-		O.item = user.get_active_hand()
-		O.s_loc = user.loc
-		O.place = "handcuff"
-		C.requests += O
-		spawn()
-			if(istype(src, /obj/item/weapon/handcuffs/cable))
-				feedback_add_details("handcuffs", "C")
-			else
-				feedback_add_details("handcuffs", "H")
-			playsound(get_turf(src), cuffing_sound, 30, 1, -2)
-			O.process()
+		var/mob/living/carbon/human/H = C
+		if (!H.has_organ_for_slot(slot_handcuffed))
+			to_chat(user, "<span class='danger'>\The [C] needs at least two wrists before you can cuff them together!</span>")
```

Comment:
"The Angelite Entyshak needs at least two wrists before you can cuff them together!"
Remove the "the" and note that this is a balance change - assistants like angelite used to take only one hand in order to not get cuffed.